### DISCUSSION TITLE
Navigationmenu move to top

### DISCUSSION THREAD

**Discussion by maka1000** [requarks/wiki#3665] [Error / Bug Report]
**Describe the bug**
When the Navigation Menu in Frontend is large and the user scroll to a specific menu item and click at this, the Navigation move back to top instead stay at same position. So the user need always do scroll down again in navigation menu.

**To Reproduce**
1. Go to your wikijis with a large menu
2. Scroll the menu to bottom and click on a menu item there
3. Then it load this page and you see, that the menu go back to top, instead hold the position. 

**Expected behavior**
The menu should not move to top. That is very circumstantial and a fast and simple navigation is not possible.

**Comment by samanhappy** [comment]
Is this issue resolved ? My deployment still not works.

**Reply by amigood** [reply]
Following. Is this a bug or a missing feature?
